libdnf/dnf-context.cpp

```diff
@@ -184,8 +186,14 @@ enum {
 };
 
 static std::string pluginsDir = DEFAULT_PLUGINS_DIRECTORY;
+static std::set<std::string> pluginsEnabled;
+static std::set<std::string> pluginsDisabled;
```

Comment:
@lukash 
All context instancies share the lists of enabled/disabled plugins. That's why the variables are global.

I can change the approach and store these lists into each context instance. It will be more flexible. But I'm not sure if we want to add this flexibility to the context API. Anyway there is another function "dnf_context_set_plugins_all_disabled()". And it globaly disables/enables all plugins. And it must be global, because it modofies configuration that is global (shared by all context instancies).